src/Tizen.Account.FidoClient/Tizen.Account.FidoClient/UafClient.cs

```diff
@@ -159,7 +163,13 @@ public static async Task<UafResponse> ProcessRequestAsync(UafMessage uafMessage,
             }
 
             TaskCompletionSource<UafResponse> tcs = new TaskCompletionSource<UafResponse>();
-            Interop.UafClient.FidoUafResponseMessageCallback cb = (int errorCode, string uafResponseJson, IntPtr userData) =>
+
+            int id;
+            lock (_callbackIdLock) {
+                id = _callbackId++;
+            }
+
+            _ResponseMessageCbMap[id] = (int errorCode, string uafResponseJson, IntPtr userData) =>
```

Comment:
I don't recommend to make callback delegate for every request, just make a one static callback delegate and make a TaskCompletionSource object for each request

you can refer this implementation https://github.com/Samsung/TizenFX/pull/1047